### PR TITLE
LPS-173394 SchemaVersion increase

### PR DESCRIPTION
**Pull request by devpedrotavares**
Follow-up: https://github.com/liferay-objects/liferay-portal/pull/2233

Related ticket: https://liferay.atlassian.net/browse/LPS-173394

Note for reviewer: in the Upgrade startup we get an exception (see photo). In spite of that, the Upgrade executes successfully and, after testing, I did not find any bugs. 
This is probably happening because there is an attempt to deploy the ObjectDefinition before the new Portlet (with new ID) is generated. 

error-in-upgrade

### CODE REVIEW

**Comment by devpedrotavares** [comment]
Rebased and fixed minor conflict

File: modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java [hunk 1]
```diff
@@ -96,8 +96,9 @@ public String getPortletId() {
 			throw new UnsupportedOperationException();
 		}
 
-		return "com_liferay_object_web_internal_object_definitions_portlet_" +
-			"ObjectDefinitionsPortlet_" + getObjectDefinitionId();
+		return StringBundler.concat(
+			"com_liferay_object_web_internal_object_definitions_portlet_",
+			"ObjectDefinitionsPortlet_", getCompanyId(), "_", getName());
```
[the remark this answers is not in the document]
Comment (murilostodolni) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java:101]: Could you change `"_"` to `StringPool.UNDERLINE`, please?

[the remark this answers is not in the document]
Comment (murilostodolni) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/model/impl/ObjectDefinitionImpl.java:101]: If you search for `portlet_ObjectDefinitionsPortlet_` in portal, we need to change for new pattern in `.testcase` and `.json` in `site-initializer-raylife-ap`.

**Review by murilostodolni — CHANGES_REQUESTED** [review]
Hey @devpedrotavares. I finished my review, let me know if you have any questions.

**Comment by murilostodolni** [comment]
@carolmariaabb could you review this PR before Pedro make the changes required, please? This PR are related with this [thread](https://liferay.slack.com/archives/C026UR5976G/p1677790862449189).

**Comment by murilostodolni** [comment]
FYI: About the error in title of PR, is related with this [ticket](https://liferay.atlassian.net/browse/LPS-169398).

**Pull request closed without merging**